Week2/5_round_numbder.js

```diff
@@ -0,0 +1,6 @@
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let BiggerNum = Math.max(a,z);
+console.log(BiggerNum);
```

Comment:
nice! Please use lowercase for your variablename, so `biggerNum` instead of `BiggerNum`, see [naming conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)